config/settings.yml

```diff
@@ -13,6 +13,7 @@ default: &default
     allow_distribution_export: false
     realtime_distributions: true
     google_analytics: true
+    allow_response_uuid_login: false
```

Comment:
do we have documentation somewhere for these options? Perhaps also update it there

Reply:
Good idea. I'll add a section on feature toggles to the readme.

Reply:
Done.